### DISCUSSION TITLE
How can you check a Bitcoin wallet address to see who it really belongs to?

### DISCUSSION THREAD

**Discussion by 2jumper3** [trustwallet/wallet-core#3216] [Q&A]
Hi all! 
We are using wallet core in our mobile app, thanks a lot for it! 
Can't find check option to resolve it. 
How can I check a Bitcoin wallet address to see who it really belongs to using WalletCore in iOS ?
Thanks a lot for help!

**Comment by weixuefeng** [comment]
what is 'who''s mean? privateKey? user?
